src/components/EasyDropdown/EasyDropdown.tsx

```diff
@@ -1,30 +1,63 @@
-import React, { useCallback, useMemo, useState } from 'react';
-import PropTypes from 'prop-types';
+import React, { useCallback, useMemo, useState, ReactElement } from 'react';
 import classNames from 'classnames';
 
 import Button from '../Button';
 import Dropdown from '../Dropdown';
 
 import styles from './EasyDropdown.module.css';
+import { MenuProps } from '../Dropdown/Menu/Menu';
+import { DropdownProps } from '../Dropdown/Dropdown';
+import { ItemProps, ButtonElementProps } from '../Dropdown/Menu/Item/Item';
 
-const EasyDropdown = ({
-  children: initChildren,
-  className,
-  defaultIsOpen,
-  disabled,
-  isOpen: controlledIsOpen,
-  menuItems,
-  menuProps,
-  onToggle: parentOnToggle,
-  toggleProps,
+export type MenuItem = ItemProps & {
+  /** Children will be rendered instead of the label, if provided */
+  children?: React.ReactNode;
+  /** If you provide group IDs, the menu items will be grouped with dividers between them. */
+  group?: string;
+  /** This will be the array key and the fallback contents */
+  label: string;
+  onClick?: () => void;
```

Comment:
You might be able to omit `children` and `onClick` from this definition, since those types should be defined already in the `ItemProps` (composed from `ButtonProps`, etc.)

Reply:
removed.. tested in storybook and local dev environment and everything works as expected.